package.json

```diff
@@ -59,7 +56,7 @@
       },
       {
         "path": "./dist/cld-video-player.light.min.js",
-        "maxSize": "145kb"
+        "maxSize": "150kb"
```

Comment:
i am just curious why we had to change size here? i mean all your changes seems dev changes?

Reply:
I was surprised as well, I guess some infra/backend caused the build to slightly increase in size (it was a very small increase)